### PR TITLE
Upload lab 5

### PR DESCRIPTION
**Pull request by viktor-kindrat**
A pull request to upload lab 5 from algos

### CODE REVIEW

**Review by Karpde — CHANGES_REQUESTED** [review]
Yon need to rewrite __repr__ method.
You need to add __str__ method.

**Comment by Karpde** [comment]
The __repr__() should return a more information-rich, or official, string representation of an object. This method is called by the built-in repr() function. If possible, the string returned should be a valid Python expression that can be used to recreate the object.

**Comment by viktor-kindrat** [comment]
> The **repr**() should return a more information-rich, or official, string representation of an object. This method is called by the built-in repr() function. If possible, the string returned should be a valid Python expression that can be used to recreate the object.

FIXED ✅

**Comment by viktor-kindrat** [comment]
> Yon need to rewrite **repr** method. You need to add **str** method.

FIXED ✅

**Review by Karpde — APPROVED** [review]
Good job!

File: lab5.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        Smartphone(
+            "Poco",
+            price=500,
+            phone_numbers=["+380985647569", "+380686513279"],
+            memory=32,
+            battery_capacity=2000
+        ),
+    ])
+
+    for el in store.sort_by_price():
```
Comment (zveres-ss) [inline lab5.py:112]: what el var name means?

Comment (viktor-kindrat) [reply lab5.py]: El variable in this case means a phone inctance in list. Varialbe was renamed to more clear varinat ✅

**Pull request closed without merging**